docs/stories/stories.md

```diff
@@ -0,0 +1,79 @@
+# COGS User Stories
+
+## Roles:
+User
+Project owner
+Principal Investigator
+Project Authoriser
+Attribution Authoriser
+RSE
+System Administrator
+
+## Stories
+### Project creation
+- Users with valid, non-student institutional email addresses can create new projects as the project owner
+- Users with student or non-institutional email address cannot create new projects
+- New projects are created with a default priority level and are not active
+- Projects do not have a start or end date (start or end dates are specified on allocations, users and attributions)
+- New projects must have a name, description, project owner and institution
+ 
+### Project administration
+- Principal investigators can associate new/existing users to an existing projects by email address
+- Associating users to a project should fail after a specified number of user has been reached (to avoid excessive disk space to a project)
+- The associated user limit should be set on a per-project basis and have a per-institution limit
+- Users associated with a project should be required to click a link in a confirm email before being added
+- Project owner can request a raised priority level if priority criterion is met (requests are sent to a system administrator to handle manually)
+
+### User creation
+- Anyone with an institutional email address should be able to sign up for a user account
+- Anyone with a pre-approved (non-institutional) email address can create an account
+- Users should have a name, email address, institution, college/department, orcid, scopus url and homepage
+- Users from Swansea university must have a Cronfa URL
+- Users should be automatically removed from the system when they leave university
```

Comment:
Such Identity Management is carried out by the universities already so we latch on to that.  COGS login will be denied when the university removes the Shibboleth auth, plus we will deploy scripts to disable the cluster users based on queries to university directory services.